service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -76,9 +78,20 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStartEvent(
 
         return AboutToStartOrSubmitCallbackResponse.builder()
             .data(data)
+            .errors(validate(mapper.convertValue(data, CaseData.class)))
             .build();
     }
 
+    private List<String> validate(CaseData caseData) {
+        ImmutableList.Builder<String> builder = ImmutableList.builder();
```

Comment:
```suggestion
        ImmutableList.Builder<String> errors = ImmutableList.builder();
```

Reply:
I am not sure it is worth wait for another build especially that context is very narrow here.